src/postgres-data/PsSource.ts

```diff
@@ -50,7 +51,18 @@ export default class PsSource implements StoreDataSourceStrategy {
     }
     next({ ...state, ...this.data, ...this.utilities })
   }
-  afterAction: StoreDataSourceStrategy['afterAction'] = (state, setState) => {
+  afterAction: StoreDataSourceStrategy['afterAction'] = (
+    state,
+    prev,
+    setState
+  ) => {
+    //  please check if that's equality check is good enough
```

Comment:
please remove that comment, it was needed during the discussion

Reply:
done